e2e/scripts/embed-and-install.sh

```diff
@@ -62,8 +63,8 @@ wait_for_memcached_pods() {
         sleep 5
         counter=$((counter+1))
         echo "Waiting for memcached pods to be ready"
-        ready=$(kubectl get pods -n helmvm | grep  Running | grep -c memcached)
-        kubectl get pods -n helmvm 2>&1
+        ready=$(kubectl get pods -n helmvm | grep  Running | grep -c memcached || true)
+        kubectl get pods -n helmvm 2>&1 || true
```

Comment:
why does this fail?

Reply:
That should not. When it started to flake in other part I added `|| true` in every place where to work or not is optional.